core/backend/eslint.config.js

```diff
@@ -0,0 +1,20 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+
+module.exports = [
+  {
+    files: ["**/*.ts"],
```

Comment:
why are some file globs `**/*.ts` and some `src/**/*.ts` ?
shouldnt they be consistent?

Reply:
Yes, I think we can just go with `**/*.ts`